tiledb/ml/models/sklearn.py

```diff
@@ -139,6 +119,24 @@ def _create_array(self):
                 )
                 raise error
 
+    def _update_array_properties(self):
+        file_properties = {}
+
+        file_properties[
+            FilePropertyName_ML_FRAMEWORK
+        ] = "SKLEARN"
+
+        # This should be an argument to this function
+        file_properties[
+            FilePropertyName_STAGE
+        ] = "STAGING"
+
```

Comment:
Why are the keys of this dict global variables while the values are hardcoded? They are not referenced anywhere else, I'd hardcode the keys too (assuming they are indeed constants and not WIP code).

Btw I'd inline the whole function, it's just one call without the visual noise:
```
tiledb.cloud.array.update_file_properties(
    uri=self.tiledb_uri,
    file_type=FILETYPE_ML_MODEL,
    file_properties={
        "ML_FRAMEWORK": "SKLEARN",
        "STAGE": "STAGING",
    }
)